### PR TITLE
Adds the hat of Doug Dimmadome, owner of the Dimmsdale Dimmadome to the game.

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑
rscadd: For half a million credits you can now purchase the hat of Doug Dimmadome, owner of the Dimmsdale Dimmadome!
/🆑

### CODE REVIEW

**Comment by Nickvr628** [comment]
The same Doug Dimmadome, owner of the Dimmsdale Dimmadome where they're showing Crash Nebula?

**Comment by fluffe9911** [comment]
On ice?

**Comment by Nickvr628** [comment]
Yeah!

**Comment by fluffe9911** [comment]
Thats right!

**Comment by drderp3635** [comment]
Sprites?

**Comment by fluffe9911** [comment]
check icon bot
https://camo.githubusercontent.com/26564af70bded0b58d461f0efea52ccc4590085c/687474703a2f2f69636f6e64696666626f742e64657874726173706163652e6e65743a383038302f46696c65732f3132303936363333362f3139343832373330312f312f61667465722e706e67

**Comment by oklews** [comment]
you should instantly explode if you wear this on the bike, or in any form of car

File: yogstation/code/modules/cargo/cargo_packs.dm [hunk 1]
```diff
@@ -173,3 +173,11 @@
 			continue
 		crate_value -= I.cost
 		new I.item(C)
+
+/datum/supply_pack/misc/milliondollarhat
+	name = "Half Off Million Dollar Hat"
+	desc = "Contains a hat that defies both logic and fashion sense. Now half off!"
+	cost = 500000
```
Comment (Amelia0010) [inline yogstation/code/modules/cargo/cargo_packs.dm:180]:
```
rscadd: For 1 million credits you can now purchase the hat of Doug Dimmadome, owner of the Dimmsdale Dimmadome!
```

```
	cost = 500000
```

image

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by AsV9** [comment]
merging this so fluffe will stop bothering me about merging this

**Pull request merged**